src/Data/Uniques/Special/Generated.lua

```diff
@@ -559,6 +559,12 @@ function buildForbidden(classNotables)
 				index = index + 1
 			end
 		end
+		table.insert(forbidden[name], "League: Siege of the Atlas")
```

Comment:
This makes the wording become "Exclusive to", and Siege isn't the league, so it'll be available after this league too.

We should remove these lines so they match other boss drops like Watcher's Eye, Mark of the _____, Solstice Vigil, etc. (Goes for all the uniques in this PR)
![image](https://user-images.githubusercontent.com/1209372/153124763-91d68660-57ac-4c29-ba49-9990b2b58b0c.png)